static/js/lightbox.js

```diff
@@ -271,6 +271,8 @@ exports.initialize = function () {
         e.preventDefault();
         // prevent the message compose dialog from happening.
         e.stopPropagation();
+        // opening a new image, so set is_open to false, if it was true.
+        is_open = false;
```

Comment:
I would expect that `is_open` would have been set to false inside `lightbox_close_overlay` (the registered handler for closing the overlay).  Can you investigate why that isn't happening?  I think that's the actual bug here, and this change might just be papering over it.

Reply:
Answered here: https://chat.zulip.org/#narrow/stream/9-issues/topic/exiting.20images.20by.20back.20button.20prevents.20them.20from.20opening.2E/near/1063712